src/SFML/Window/Android/InputImpl.cpp

```diff
@@ -166,7 +166,8 @@ void setVirtualKeyboardVisible(bool visible)
 ////////////////////////////////////////////////////////////
 bool isMouseButtonPressed(Mouse::Button button)
 {
-    ALooper_pollAll(0, nullptr, nullptr, nullptr);
+    while (ALooper_pollOnce(0, nullptr, nullptr, nullptr) != ALOOPER_POLL_CALLBACK)
+        ;
```

Comment:
Shouldn't this be the other way around?  
For as long as you get `ALOOPER_POLL_CALLBACK` you should continue processing, every other result should lead to a break.

What's curious is that the official NDK sample, doesn't do it in a loop: https://github.com/android/ndk-samples/pull/1008/files